src/modules/3box-aragon.js

```diff
@@ -62,12 +62,14 @@ export class Profile {
     const publicProfile = this.boxState.opened
       ? await this.unlockedBox.public.all()
       : await Box.getProfile(this.ethereumAddress)
-
     return publicProfile
   }
 
   getPublic = async () => {
     const publicProfile = await this._getPublic()
+    if (!Object.keys(publicProfile).length) {
```

Comment:
Why is this necessary?

Reply:
use3Box hook gets Profile like this:

`const publicProfile = await profile.getPublic()`

getPublic() downloads profile and adds github and twitter keys to it. If Profile is empty - return early, so the fields are not added, and the hook knows there is no profile. There are probably two alternatives here: either check if getPublic() returns empty object, or if returns an object missing one special key that should be there and if there isn't it means the profile is empty.